lib/src/runner.dart

```diff
@@ -106,12 +106,16 @@ class MigratorRunner extends CommandRunner<Map<Uri, String>> {
 
     if (argResults['dry-run']) {
       print('Dry run. Logging migrated files instead of overwriting...\n');
```

Comment:
Would it make sense to remove this line or make it log to stderr so that the HRX output could be piped somewhere?

Reply:
I think we actually *don't* want people piping the output, since we plan to change the format to a diff at some point in the future anyway so it won't be reliably forwards-compatible.